src/components/HistoryMentorshipCardFromBot/index.tsx

```diff
@@ -1,93 +1,80 @@
 import { formatDate } from '@/helpers/formatDate';
 import { IMentorship } from '@/interfaces/mentorship.interface';
-import { ChevronRightIcon } from '@heroicons/react/outline';
-import React, { useState } from 'react';
-import classNames from 'classnames';
+import React, { Dispatch, SetStateAction } from 'react';
+import { SELF_HISTORY } from '@/config/Routes';
+import Link from 'next/link';
+import { isAdmin } from '@/helpers/IsAdmin';
+import { useNextAuthSession } from '@/hooks/useNextAuthSession';
+import { deleteMentorship } from '@/services';
 
 interface IMentorshipCard {
   mentorship: IMentorship;
-}
-
-interface CardEventTarget extends EventTarget {
-  id?: string;
-}
-
-interface CardMouseEvent extends React.MouseEvent<HTMLElement> {
-  target: CardEventTarget;
+  setLoading: Dispatch<SetStateAction<boolean>>;
+  setMentorships: (id: string) => void;
 }
 
 const MentorshipCardFromBot: React.FC<IMentorshipCard> = ({
   mentorship: {
     id,
     mentee_id,
-    mentee_username_discord,
-    mentor_username_discord,
+    mentee_name,
+    mentor_name,
     mentorship_create_date,
     mentor_id,
   },
+  setLoading,
+  setMentorships,
 }) => {
-  const [isOpen, setIsOpen] = useState(false);
-  const date = String(mentorship_create_date);
+  const [session, loading] = useNextAuthSession();
+
+  const onDeleteMentorship = async () => {
```

Comment:
This function should be executed after a modal confirmation, I removed one record by mistake 😓